app/views/charges/new.html.erb

```diff
@@ -6,13 +6,12 @@
       </div>
     <% end %>
     <label class="amount">
-      <span>Amount: <%= @amount %></span>
+      <span>Amount: <%= current_user.donations.last %></span>
```

Comment:
This will show the entire donations object - don't we want the amount from the donations object?